src/homepage/tests/test_user.py

```diff
@@ -3,13 +3,19 @@
 from django.core.exceptions import ObjectDoesNotExist
 from rest_framework.authtoken.models import Token
 
+
 class UserTest(TestCase):
     def setUp(self):
-        self.test_user = User.objects.create_user('TestUser', 'test@sanaprotocolbuilder.me', 'testpassword')
+        self.test_user = User.objects.create_user(
+            'TestUser',
+            'test@sanaprotocolbuilder.me',
+            'testpassword'
+        )
         self.test_user.save()
 
     def test_user_has_token(self):
         try:
             token = Token.objects.get(user=self.test_user)
+            self.assertFalse(token is None)
```

Comment:
I think it's easier to read if it's `self.assertTrue(token is not None)` to avoid double negatives